dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/Standard/Steam/GovSteamEU.mo

```diff
@@ -0,0 +1,336 @@
+    Dialog(tab = "Emergency control"));
+  parameter Types.AngularVelocityPu OmegaOmegaMaxPu "Upper limit for the speed governor frequency correction in pu (base SystemBase.omegaNom), typical value = 0.1" annotation(
+    Dialog(tab = "Islanded control"));
+  parameter Types.AngularVelocityPu OmegaOmegaMinPu "Lower limit for the speed governor frequency correction in pu (base SystemBase.omegaNom), typical value = -1" annotation(
+    Dialog(tab = "Islanded control"));
+  parameter Types.ActivePowerPu PBaseMw "Base power of the machine in [MW, MVA], typical value = PNomTurb" annotation(
+    Dialog(tab = "General"));
+  parameter Types.ActivePowerPu PGenBaseMw = SystemBase.SnRef "Base power for measured electrical power in [MW, MVA], typical value = SystemBase.SnRef" annotation(
+    Dialog(tab = "General"));
+  parameter Types.ActivePowerPu PMaxPu "Maximal active power of the turbine in pu (base PNomTurb), typical value = 1" annotation(
+    Dialog(tab = "Grid connected control"));
+  parameter Types.PerUnit PRhMaxPu "Maximum low pressure limit in pu (base PNomTurb), typical value = 1.4" annotation(
+    Dialog(tab = "Turbine / boiler"));
+  parameter Types.PerUnit SimxPu "Intercept valves transfer limit in pu (base PNomTurb), typical value = 0.425" annotation(
+    Dialog(tab = "Emergency control"));
+  parameter Types.Time tB "Boiler time constant in s, typical value = 100" annotation(
+    Dialog(tab = "Turbine / boiler"));
+  parameter Types.Time tVHp "Control valves servo time constant in s, typical value = 0.1" annotation(
+    Dialog(tab = "Valves"));
+  parameter Types.Time tDp = 1e-9 "Power controller derivative time constant in s, typical value = 1e-9" annotation(
+    Dialog(tab = "Grid connected control"));
+  parameter Types.Time tEn "Electro hydraulic transducer time constant in s, typical value = 0.1" annotation(
+    Dialog(tab = "Grid connected control"));
+  parameter Types.Time tF = 1e-9 "Frequency transducer time constant in s, typical value = 1e-9" annotation(
+    Dialog(tab = "Grid connected control"));
+  parameter Types.Time tHp "High pressure turbine time constant in s, typical value = 0.31" annotation(
+    Dialog(tab = "Turbine / boiler"));
+  parameter Types.Time tIp "Power controller integral time constant in s, typical value = 2" annotation(
+    Dialog(tab = "Grid connected control"));
+  parameter Types.Time tVLp "Intercept valves servo time constant in s, typical value = 0.15" annotation(
+    Dialog(tab = "Valves"));
+  parameter Types.Time tLp "Low pressure turbine time constant in s, typical value = 0.45" annotation(
+    Dialog(tab = "Turbine / boiler"));
+  parameter Types.Time tOmega "Speed transducer time constant in s, typical value = 0.02" annotation(
+    Dialog(tab = "Islanded control"));
+  parameter Types.Time tFp = 1e-9 "Power controller derivative numerical approximation time constant in s, typical value = 1e-9" annotation(
```

Comment:
Could you remove the 1e-9 values, I had problems with GovCt2 in dynawo so I redruced the values to 1e-6, I prefer not having a default value